linkcheck/htmlutil/linkparse.py

```diff
@@ -104,18 +104,15 @@ class TagFinder (object):
     def __init__ (self):
         """Initialize local variables."""
         super(TagFinder, self).__init__()
-        # parser object will be initialized when it is used as
-        # a handler object
-        self.parser = None
 
-    def start_element (self, tag, attrs):
```

Comment:
i don't know how we make note of that, but we should definitely keep in mind that the next release should change major version as per Semver. I know there's probably a billion other changes like this that warrant that, but it's just something i figured i would bring up.

linkchecker 10! :)

Reply:
I was thinking the same. Maybe even some 9.9's (tags only? because `pip install` looks easy  - haven't tried), some early testers would be great.

Reply:
i would do 10.0rc1 maybe? at least if we're going to do a test release... let's move this to another ticket: #359 :)